### PR TITLE
Add BEGINNER/ADVANCED structure

### PR DESCRIPTION
**Pull request by BTCparadigm**
If the [idea](https://github.com/zkSNACKs/WalletWasabi/discussions/6740#discussioncomment-1840029) mentioned in [#6740](https://github.com/zkSNACKs/WalletWasabi/discussions/6740) is accepted, this PR creates few new files and folders to the 2.0 docs.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[WIP]
+
+This is Wasabi Wallet 2.0's open source documentation. 
+
+Even though we have tried to make our new software UI as intuitive as possible, we should still describe the functions of our wallet and explain the magic under the hood, so users can make better decisions regarding their financial privacy.
```
Comment (nopara73) [inline docs/README.md:5]: software's

Comment (nopara73) [inline docs/README.md:5]: - possession
- Don't use the word "new", it makes it obsolate fast

```suggestion
Even though we have tried to make our software's UI as intuitive as possible, we should still describe the functions of our wallet and explain the magic under the hood, so users can make better decisions regarding their financial privacy.

```

**Review by nopara73 — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (BTCparadigm) [reply docs/README.md]: What do you mean by "possession"?

**Comment by MarnixCroes** [comment]
NACK
I thought we agreed to not separate into _beginner_ and _advanced_ at "main page".
f.e.: `send`can have _advanced_ aspects aswell

**Comment by BTCparadigm** [comment]
I understood this idea is still WIP and the titles can be changed. I actually like Bitcoin-Pleb and Super Shadowy Coder levels even more than beginner/advanced :D
I dont know if there has been any other practical idea for the structure.

**Review by yahiheb — APPROVED** [review]
LGTM

**Review by nopara73 — CHANGES_REQUESTED** [review]
Concept NACK. It make no sense to make significant structural changes to the docs. This repository should be deleted.

**Pull request closed without merging**